build.ps1

```diff
@@ -62,7 +62,7 @@ Param(
     [switch] $clean,
     [switch] $build,
     [switch] $unittests,
-    [switch] $prtests,
```

Comment:
I think PR Tests was a better name for how we're using it. They aren't BVTs, strictly speaking. Our integration build and nightly builds run all the tests that validate the build.

Reply:
I don't mind changing it back to PR tests.